android/res/values/themes-base.xml

```diff
@@ -154,7 +154,6 @@
 
     <item name="alertDialogTheme">@style/MwmTheme.Night.AlertDialog</item>
     <item name="windowBackgroundForced">@color/bg_window_night</item>
-    <item name="cardFrame">@drawable/card_frame_night</item>
```

Comment:
These pngs should be deleted if they're not used.

Reply:
I wasn't sure if they were still used, but I didn't realise android studio has safe delete! Removed them now 👍